roles/add_livemigration_user/tasks/main.yml

```diff
@@ -24,25 +27,26 @@
       ssh_key_file: .ssh/id_rsa
       ssh_key_passphrase: ""
       force: false
-  - name: create livemigration .ssh directory
+  - name: create livemigration ssh directory
     file:
-      path: "/home/{{ livemigration_user }}/.ssh"
+      path: "{{ ssh_path }}"
```

Comment:
In my opinion, we should rather change the ssh configuration on Yocto to use the regular one. The change was made due to the readonly feature before we added the /home directory in overlay.